test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -696,6 +696,7 @@ def test_pip_check(image):
 
 @pytest.mark.usefixtures("sagemaker", "huggingface")
 @pytest.mark.model("N/A")
+@pytest.mark.skipif(not is_dlc_cicd_context(), reason="This relies on buildspec env variables being properly set")
 def test_cuda_paths(gpu):
```

Comment:
seems like a candidate for quick-checks. Do we need to run this in mainline at all? PR context seems enough - and I'm not convinced we need to enforce this convention anymore

Reply:
If we also need this in mainline, then we also need to pass FRAMEWORK_BUILDSPEC_FILE to the subsequent jobs in mainline context. We can do this with secondary artifacts or other methods. But to me this seems a bit much for accommodating one test